vcd/resource_vcd_cloned_vapp_test.go

```diff
@@ -0,0 +1,189 @@
+//go:build vapp || ALL || functional
+
+package vcd
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/terraform"
+)
+
+func TestAccVcdClonedVApp(t *testing.T) {
+	preTestChecks(t)
+	var vappFromTemplate = "TestAccClonedVAppFromTemplate"
+	var vappFromVapp = "TestAccClonedVAppFromVapp"
+	var vappDescription = "Test cloned vApp from Template"
+	vappTemplateName := "Test-small-3VM"
+	firstVmName := "firstVM"
+	//secondVmName := "secondVM"
+	//thirdVmName := "thirdVM"
```

Comment:
Is the comment still needed?

Reply:
No. In fact, even `firstVmName` was practically unused, since I use `vm_list` to get the actual VM name. Now removed all three.
Thanks